airflow/api_fastapi/auth/managers/simple/ui/dev/index.html

```diff
@@ -3,7 +3,7 @@
 <html lang="en" style="height: 100%">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/png" href="http://localhost:5174/public/pin_32.png" />
+    <link rel="icon" type="image/png" href="http://localhost:5174/api/v2/pin_32.png" />
```

Comment:
Static files should not be under the API prefix.

Reply:
Ash is right. Maybe no prefix is a good alternative for static files.

Reply:
Let's do this refactor in a follow up too.

Reply:
So this is the dev ui - in the compiled ui they come from `/static` already. I think this is okay as-is.